### PR TITLE
use honeycombio/mongodbtools/logparser

### PR DESCRIPTION
**Pull request by toshok**
pretty minor changes to retarget parsers and handle some additional processing (parsing timestamps by mongodb-supported formats).

The bulk of this work is in the honeycombio/mongodbtools repo.

### CODE REVIEW

File: parsers/mongodb/mongodb.go [hunk 1]
```diff
@@ -28,34 +38,75 @@ type MongoLineParser struct {
...
 	return nil
 }
 
+func (p *Parser) parseTimestamp(values map[string]interface{}) (time.Time, error) {
+	now := p.nower.Now()
+	timestamp_value, ok := values["timestamp"].(string)
+	if ok {
+		var err error
+		for _, f := range timestampFormats {
+			timestamp, err := time.Parse(f, timestamp_value)
```
Comment (maplebed) [inline parsers/mongodb/mongodb.go]: change to
```
var timestamp time.Time
timestamp, err = time.Parse(f, timestamp_value)
```

File: parsers/mongodb/mongodb.go [hunk 2]
```diff
@@ -28,34 +38,75 @@ type MongoLineParser struct {
...
+func (p *Parser) Init(options interface{}) error {
+	p.conf = *options.(*Options)
 	p.nower = &RealNower{}
 	p.lineParser = &MongoLineParser{}
 	return nil
 }
 
+func (p *Parser) parseTimestamp(values map[string]interface{}) (time.Time, error) {
+	now := p.nower.Now()
+	timestamp_value, ok := values["timestamp"].(string)
```
Comment (maplebed) [inline parsers/mongodb/mongodb.go]: maybe pull "timestamp" out in to a constant "timestampFieldName" for typo-protection and magic-identification?

File: parsers/mongodb/mongodb.go (hunk 1 above)
Comment (toshok) [inline parsers/mongodb/mongodb.go]: :+1:

**Comment by maplebed** [comment]
🚢

**Pull request merged**